src/main/java/common/Common.java

```diff
@@ -1,22 +1,10 @@
 package common;
 
-import org.json.simple.JSONArray;
-import org.json.simple.parser.JSONParser;
-import org.json.simple.parser.ParseException;
-
 import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 
 public class Common {
-    public static JSONArray getJsonData(String pathFileJson) throws IOException, ParseException {
-        JSONParser jsonParser = new JSONParser();
-        FileReader reader = new FileReader(pathFileJson);
-        Object obj = jsonParser.parse(reader);
-        return (JSONArray) obj;
-    }
 
     public static String plusDayFromNow(long daysToAdd, String formatDay) {
```

Comment:
formatDay -> dateFormat

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.